Lab_2/Syntax.in

```diff
@@ -0,0 +1,43 @@
+Sintactical rules: 
+
+program = decllist ";" stmtlist 
```

Comment:
declarations only available at the beginning of the program?

Reply:
Yes, this is how I want the program to work